R/compute_duration.R

```diff
@@ -1,4 +1,4 @@
-#' Derive Duration
+#' Compute Duration
 #'
 #' Derives duration between two dates, e.g., duration of adverse events,
```

Comment:
```suggestion
#' Computes duration between two dates, e.g., duration of adverse events,
```

Reply:
Since it just the duration between one event - maybe we should just say an adverse event??

Reply:
![image](https://user-images.githubusercontent.com/10111024/162791894-a09cb3d1-e900-4a71-98d8-7b1013b876ce.png)

@bundfussr Do you think anything else should be included to help differentiate the difference between the dervive and compute derivation functions?  I was looking at the programming strategy and there is this reference compute functions being used in derived functions.  Maybe we should put something of the effect in the `@details` section?